share/qtcreator/sfdk/modules/50-testing-mb2.json

```diff
@@ -32,7 +32,7 @@
         },
         {
             "name": "check",
-            "trSynopsis": "--list-suites\n[-l|--levels [+|-]<level>...] [-s|--suites [+|-]<suite>...] [<artefac>]...",
+            "trSynopsis": "--list-suites\n[-l|--levels [+|-]<level>...] [-s|--suites [+|-]<suite>...] [--] [<artefac>]...",
```

Comment:
Could you add the missing 't' to the word 'artefact' while at it? Otherwise, LGTM

Reply:
The commit is on the bottom, I will push another commit after merging all the PRs.

Reply:
Added to PR #444